src/foundry/foundry.js/clientDocuments/scene.d.ts

```diff
@@ -8,9 +18,450 @@ declare global {
    * @see {@link documents.Scenes}            The world-level collection of Scene documents
    * @see {@link applications.SceneConfig}    The Scene configuration application
    *
-   * @param data - Initial data provided to construct the Scene document
    */
-  class Scene extends ClientDocumentMixin(foundry.documents.BaseScene) {}
+  class Scene extends ClientDocumentMixin(foundry.documents.BaseScene) {
+    /**
+     * @param data - Initial data provided to construct the Scene document
+     */
+    constructor(
+      data: ConstructorParameters<typeof foundry.documents.BaseScene>[0],
+      context?: ConstructorParameters<typeof foundry.documents.BaseScene>[1]
+    );
+
+    /**
+     * Determine the canvas dimensions this Scene would occupy, if rendered
+     * @defaultValue `{}`
+     */
+    dimensions: ReturnType<typeof Canvas.getDimensions> | {};
+
+    /**
+     * Track whether the scene is the active view
+     */
+    protected _view: this['data']['active'];
+
+    /**
+     * Track the viewed position of each scene (while in memory only, not persisted)
+     * When switching back to a previously viewed scene, we can automatically pan to the previous position.
+     * @remarks This is intentional public because it is used in Canvas._initializeCanvasPosition() and Canvas.pan()
```

Comment:
```suggestion
     * @remarks This is intentionally public because it is used in Canvas._initializeCanvasPosition() and Canvas.pan()
```
Also might make sense to document the `defaultValue`?